srcdoc/configure/h2olog.mt

```diff
@@ -0,0 +1,85 @@
+? my $ctx = $main::context;
+? $_mt->wrapper_file("wrapper.mt", "Configure", "Using h2olog for Tracing")->(sub {
+
+<p>h2olog is a <a href="https://www.kernel.org/doc/html/latest/bpf/index.html">BPF</a> (<a href="https://www.kernel.org/doc/Documentation/networking/filter.txt">kernel doc</a>) backed HTTP request tracing tool for the <a href="https://github.com/h2o/h2o">H2O</a> server.
+h2olog can also be used to log <a href="https://en.wikipedia.org/wiki/QUIC">QUIC</a> events for <a href="https://en.wikipedia.org/wiki/Transport_layer">transport layer</a> observation.
+See <a href="#tracing-quic-events">Tracing QUIC events</a> for how.</p>
+
+<h2 id="installing-from-source">Installing from Source</h2>
+
+<p>See <a href="#requirements">requirements</a> for build prerequisites. If dependencies are satisfied, <code>-DWITH_H2OLOG=on</code> will be set and the <code>h2olog</code> target will be configured.</p>
+<p>If you have <code>BCC</code> installed to a non-standard path, use <code>pkg-config</code> for <code>cmake</code>.
+
+<?= $ctx->{code}->(<<'EOT')
+$ PKG_CONFIG_PATH=/path/to/bcc/lib/pkgconfig cmake [options]
+EOT
+?>
+
+<h2 id="requirements">Requirements</h2>
+
+<h3>For building h2olog</h3>
+<ul>
+<li>C++11 compiler</li>
+<li>CMake (&gt;= 3.8) for generating the build files</li>
```

Comment:
I am not sure if this is true. We check the version only for setting the default.